src/imitation/scripts/common/rl.py

```diff
@@ -132,3 +133,25 @@ def make_rl_algo(
     logger.info(f"RL algorithm: {type(rl_algo)}")
     logger.info(f"Policy network summary:\n {rl_algo.policy}")
     return rl_algo
+
+
+@rl_ingredient.capture
+def load_rl_algo_from_path(
+    agent_path: str,
+    venv: vec_env.VecEnv,
+    rl_cls: Type[base_class.BaseAlgorithm],
+    rl_kwargs: Mapping[str, Any],
+    _seed: int,
+) -> base_class.BaseAlgorithm:
+    agent = serialize.load_stable_baselines_model(
+        rl_cls,
+        agent_path,
+        venv,
+        seed=_seed,
+        **rl_kwargs,
```

Comment:
What do you want to happen when `rl_kwargs` in the script does not match the `rl_kwargs` that was used during training `agent_path`? Right now we're overriding them (by default they're copied from the saved pickle file). This is reasonable -- sometimes you might want to restart training with e.g. a different learning rate -- but will sometimes cause things to fail. I think most of the failures should be obvious (e.g. you try and instantiate with a different sized policy network and things blow up), but it's possible some failures could be silent.

Reply:
I checked the source, and I think that is what I want it to behave. I want kwargs passed into `load_stable_baselines_model` to override the saved kwargs.